packages/mobx-react/src/utils/FinalizationRegistryWrapper.ts

```diff
@@ -0,0 +1,12 @@
+declare class FinalizationRegistryType<T> {
```

Comment:
Any reason to not import these utils from `mobx-react-lite`?

Reply:
Note, it's a slightly artifical, but you can `this[reactionTrackingRef] = React.createRef()`

Reply:
> Note, it's a slightly artifical, but you can `this[reactionTrackingRefSymbol] = React.createRef()`

Yes, but I don't think it's necessary. When a function component uses ref, because it can't get the component instance, the class component can directly cache this to achieve the same effect

Reply:
> Any reason not to import these utils from `mobx-react-lite`?

If you think this is more reasonable, I can modify it here. Initially, a separate file was created for decoupling between different packages

Reply:
Yes, `mobx-react` already depends on `mobx-react-lite`, it more or less only adds class support.

Reply:
i will do it